src/main/java/com/flightstats/hub/webhook/InternalWebhookResource.java

```diff
@@ -22,10 +22,7 @@
 import javax.ws.rs.core.UriBuilder;
 import javax.ws.rs.core.UriInfo;
 import java.net.URI;
-import java.util.Collection;
-import java.util.Map;
-import java.util.TreeMap;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
Looks like IntelliJ settings are out of sync. The semi-standard we've been running with is do not wildcard unless it's a huge number (mine is set to 100).